Resources/public/templates/testonearticlesview.hbt

```diff
@@ -0,0 +1,8 @@
+{{#if previous}}
+<a href="#" data-location-id="{{previous.locationId}}" class="ez-testoneview-article ez-testoneview-previous">Back to {{previous.name}}</a>
```

Comment:
Nie podoba mi się to złożenie klas. Staramy się stosować do metodyki BEM, czyli w tym przypadku bym to zrobił tak: `testone-article__link testone-article__link--previous`.